Client/sdk/game/CCarEnterExit.h

```diff
@@ -23,4 +23,6 @@ class CCarEnterExit
                                             bool bCheckIfRoomToGetIn) = 0;
     virtual int  ComputeTargetDoorToExit(CPed* pPed, CVehicle* pVehicle) = 0;
     virtual bool IsRoomForPedToLeaveCar(CVehicle* pVehicle, int iDoor, CVector* pUnknown = 0) = 0;
+
+    virtual const void GetPositionToOpenCarDoor(CVector& position, CVehicle* vehicle, unsigned int door) = 0;
```

Comment:
If its possible, could you mark this function as const?
```cpp
virtual const void GetPositionToOpenCarDoor(...) const = 0;
```
If the function doesnt throw, mark ot as `noexcept` too, please

Reply:
Compiler doesn't like this, adding noexcept though.